compiler/luci-interpreter/src/kernels/FloorDiv.test.cpp

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ * Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "kernels/FloorDiv.h"
+#include "kernels/TestUtils.h"
+
+namespace luci_interpreter
+{
+namespace kernels
+{
+namespace
+{
+
+using namespace testing;
+
+TEST(FloorDivTest, FloatSimple)
+{
+  std::initializer_list<int32_t> x_shape{2, 3};
```

Comment:
Is there a specific reason not to use `std::vector` for shapes?

Reply:
It's used here to feed both `makeInputTensor` and `ElementsAreArray` down below without need for conversion. `makeInputTensor` uses to implicitly create `Shape`, but `Shape` doesn't have `begin()`/`end()` to use it in default gmock matcher.

Reply:
Please use `vector`. It is both consistent and avoids confusion -- `initializer_list`s were designed for other purpose. It is quite unusual to see them as local variables.

Reply:
Done. Replaced with `vector` for reference shapes and `Shape`s.